src/playground/App.js

```diff
@@ -49,9 +52,39 @@ const fillOptionsDefaults = options =>
             }, {})
         });
 
+const convertLegacyOptionsToFlatConfig = (options = {}) => {
+
+    // If options object is empty, return it
+    if (isEmptyObject(options)) {
+        return options;
+    }
+
+    // If options is already a flat config, return it
+    if (typeof options.languageOptions !== "undefined" && typeof options.parserOptions === "undefined") {
```

Comment:
should also check `env` - it's away in the new config.

Reply:
This is now refactored a bit, please check the new version.